test/topology_experimental_raft/test_topology_ops.py

```diff
@@ -101,18 +101,15 @@ async def start_writes(cql: Session, concurrency: int = 3):
 
     async def do_writes(worker_id: int):
         write_count = 0
-        last_error = None
         while not stop_event.is_set():
             start_time = time.time()
             try:
                 await cql.run_async(stmt)
                 write_count += 1
             except Exception as e:
                 logger.error(f"Write started {time.time() - start_time}s ago failed: {e}")
-                last_error = e
+                raise
         logger.info(f"Worker #{worker_id} did {write_count} successful writes")
-        if last_error is not None:
```

Comment:
Was there a purpose to not raising immediately? @piodul

Reply:
Not sure there was. I guess the old behavior could be useful in determining whether the failure was persistent or transient, and in the latter case - when exactly it gets fixed. However, it's not worth it if we generate 20GB of logs as a result.